tests/unit/test_filter_test_cases.py

```
@@ -1,151 +1,181 @@
+        assert data["already_implemented"] == []
+        assert data["be_test_cases"] == ["TC-E2E-001"]
 
-    def test_handles_nonexistent_tc_file(self, tmp_path):
-        """Should raise error if TC file doesn't exist."""
-        tc_dir = tmp_path / "test_cases"
-        tc_dir.mkdir()
-        (tc_dir / "INDEX.md").write_text("# Index")
+    def test_missing_tc_file_raises(self, tmp_path):
+        (tmp_path / "test_cases").mkdir()
 
         with pytest.raises(FileNotFoundError, match="TC-MISSING-001.md not found"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)